core_lib_generator/config_collectors/database.py

```diff
@@ -83,11 +89,21 @@ def _generate_db_config(
     db_pool_pre_ping: bool = False,
 ) -> dict:
     env = {}

```

Comment:
1. separate function create URL

2. make sure config dict is declared only one time.